website/versioned_docs/version-28.1/Configuration.md

```diff
@@ -761,20 +761,26 @@ A preset that is used as a base for Jest's configuration. A preset should point
 
 For example, this preset `foo-bar/jest-preset.js` will be configured as follows:
 
-```json
-{
-  "preset": "foo-bar"
+```js
+/** @type { import('@jest/types').Config.InitialOptions } */
+module.exports = {
```

Comment:
Makes me doubt if typings and `module.exports` should be included with each example. There is a ton of examples on this page, hence adding two lines to each of them consistently would take lots of vertical space. Both, typings and `module.exports`, are already mentioned at the top of this page.

At first glance it looked like these are examples of presets. Might be useful to mentioned that presets can be typed this way? Perhaps. But seems like these are just `jest.config` examples. Not sure if this change is necessary. If there is a motivation behind, then perhaps all examples of this page should be refactored in similar fashion?

Reply:
you know... I had the same internal struggle when I first wrote this.  It's similar in lots of other docs for _any_ package with the common question "should the import be there every time at the top of the JS file example".

my thinking is that docs are there for people that don't always have everything figured out, and if they can copy-paste their way to victory, then some repetition in the docs is a worthwhile cost to pay.